src/Core/src/ImageSources/UriImageSourceService/UriImageSourceService.iOS.cs

```diff
@@ -78,7 +78,9 @@ public void CacheImage(NSData imageData, string path)
 
 			Directory.CreateDirectory(directory);
 
+#pragma warning disable CA1416 // TODO: 'NSData.Save(string, bool)' is unsupported on: 'ios' 13.0 and later
 			var result = imageData.Save(path, true);
+#pragma warning restore CA1416
```

Comment:
Not sure this is correct: https://developer.apple.com/documentation/foundation/nsdata/1408033-writetofile?language=objc

The docs show 2.0+ so I think this may be a binding issue again? Especially since the overload that is an `out NSError` is fine to use...

Reply:
You mean the annotation is incorrect? 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/45579687/165597572-a6eca84f-b267-4c49-8a23-a1802d41624b.png">

Reply:
Added to https://github.com/xamarin/xamarin-macios/issues/14619